crates/ruff_linter/src/message/diff.rs

```diff
@@ -54,9 +54,9 @@ impl Display for Diff<'_> {
 
         let message = match self.fix.applicability() {
             // TODO(zanieb): Adjust this messaging once it's user-facing
-            Applicability::Safe => "Fix",
-            Applicability::Unsafe => "Suggested fix",
-            Applicability::Display => "Possible fix",
+            Applicability::Safe => "Safe fix",
+            Applicability::Unsafe => "Unsafe fix",
+            Applicability::Display => "Display only fix",
```

Comment:
Nit: "Display-only fix"? As-is it reads as imperative to me, like an action.

Reply:
That works. Should we consider improving the name of that applicability level?

Reply:
To `DisplayOnly`?

Reply:
I'm fine with that